Proposals/xgov-192.md

```diff
@@ -0,0 +1,156 @@
+---
+id: 192
+period: 4
+title: PiPhi Network Smart Home Visualization Platform
+author: Justin Somerville <jsomerville@piphi.network>, Kelvin Sanyaolu (@KelvinSan)
+email: jsomerville@piphi.network
+discussions-to: https://forum.algorand.org/t/xgov-188-piphi-network-submission/11807
+company_name: PiPhi Network
+category: Other
+focus_area: Environmental Monitoring, Software, Developers
```

Comment:
Please refer to the error below.

 | focus_area: Environmental Monitoring, Software, Developers
   |            ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^ must be one of: `Banking`, `Defi`, `Dex`, `Gaming`, `Identity`, `Marketplace`, `Metaverse`, `NFT`, `Oracle`, `Storage`, `User Onboarding`, `IDE`, `Teal`, `Deployment`, `Libraries`, `Monitoring`, `Node`, `Education`, `Social`, `Wallet`, `Other.`

Reply:
Change has been made

Reply:
@SudoWeezy Changes have been made, please let us know if there is anything else we need to fix. Thank you